cloudify_rest_client/exceptions.py

```diff
@@ -26,9 +26,15 @@ def __init__(self, message, server_traceback=None,
         self._message = message
 
     def __str__(self):
+        message = self._message
+        if self.server_traceback:
+            traceback_list = \
+                [line for line in self.server_traceback.split('\n') if line]
```

Comment:
we only care about the last line, so how about something like...

```
_, _, last_line = self.server_traceback.strip().rpartition('\n')
if last_line:
```

Reply:
not really, usually the last line would be just empty string

Reply:
hence the `.strip()` ;)